src/token_utilities.py

```diff
@@ -21,47 +22,41 @@ def process_token(token):
     parts = token.split('.')
     if len(parts) == 3:
         try:
-            payload = decode_access_token(token)
-            unique_name = payload.get('unique_name')
+            decoded_payload = decode_access_token(parts)
+            unique_name = decoded_payload.get('unique_name')
             if not unique_name:
-                raise ValueError("Unique name not found in token payload")
+                raise ValueError('Unique name not found in token payload')
 
-            scope = payload.get('scp')
-            time = datetime.now().strftime("%H:%M:%S")
-            today = date.today().strftime("%d-%m-%Y")
+            scope = decoded_payload.get('scp')
+            time = datetime.now().strftime('%H:%M:%S')
+            today = date.today().strftime('%d-%m-%Y')
 
-            cache_key = 'user_data_' + unique_name
+            cache_key = f'user_data_{unique_name}'
             cache_data = {
-                "access_token": token,
-                "scope": scope,
-                "time": time,
-                "date": today,
-                "email_address": unique_name
+                'access_token': token,
+                'scope': scope,
+                'time': time,
+                'date': today,
+                'email_address': unique_name
             }
 
             cache.set(cache_key, cache_data)
-            access_token = cache_key, cache_data
-            return access_token
+            return {'user_data_key': cache_key, 'data': cache_data}
 
         except Exception as e:
-            opaque_token_count = cache.get(OPAQUE_TOKEN_COUNT_KEY, 0) + 1
-            cache.set(OPAQUE_TOKEN_COUNT_KEY, opaque_token_count)
-
-            opaque_key = f'opaque_{opaque_token_count}'
-            cache.set(opaque_key, token)
-            raise e  
-    else:
-        opaque_token_count = cache.get(OPAQUE_TOKEN_COUNT_KEY, 0) + 1
-        cache.set(OPAQUE_TOKEN_COUNT_KEY, opaque_token_count)
-
-        opaque_key = f'opaque_{opaque_token_count}'
-        time = datetime.now().strftime("%H:%M:%S")
-        today = date.today().strftime("%d-%m-%Y")
-        opaque_data = {
-            "opaque_token": token,
-            "time": time,
-            "date": today
-        }
-
-        cache.set(opaque_key, opaque_data)
-        return {"opaque_key": opaque_key, "data": opaque_data}
\ No newline at end of file
+            logging.error(f'Error processing access_token: {e}')
+
+    opaque_token_count = cache.get(OPAQUE_TOKEN_COUNT_KEY, 0) + 1
```

Comment:
Why are you using OPAQUE_TOKEN_COUNT?

You remove the definition from all other files, but decide to import it only in the file and use strings in other file. Be consistent, either use it everywhere o no where.